client-v2/src/components/FrontsEdit/ArticleFragmentForm.tsx

```diff
@@ -495,6 +458,35 @@ class FormComponent extends React.Component<Props, FormComponentState> {
       </FormContainer>
     );
   }
+
+  private changeImageField = (fieldToSet: string) => {
+    this.allImageFields.forEach(field => {
+      if (field === fieldToSet) {
+        this.props.change(field, true);
+      } else {
+        this.props.change(field, false);
+      }
+    });
+  };
+
+  private getImageFieldName = () => {
+    if (this.props.imageCutoutReplace) {
+      return 'cutoutImage';
+    }
+    return 'primaryImage';
+  };
+
+  private handleImageChange: EventWithDataHandler<React.ChangeEvent<any>> = (
+    e,
+    ...args: [any?, any?, string?]
+  ) => {
+    // If no image overrides are enabled, enable the default imageReplace property.
+    // This saves the user a click; adding an image without enabling would be very unusual.
```

Comment:
Small thing - I don't understand this comment. Is it that: "If no images are overriden, enable the default imageReplace property. This saves the user a click, because they no longer need to enable this to start changing the image?

Reply:
Jenny's suggestion: `Given a state where nothing is toggled, and an image is added - we would assume that the imageReplace toggle should be automatically activated, as this is the most likely reason for the user to add an image.`

Reply:
Thanks for spotting, the `no` in this comment means the reverse of what was intended! I'll rephrase. 👍

Reply:
Hopefully that's clearer now!